migrations/versions/6a38281e6fcc_user_detail_based_tables.py

```diff
@@ -0,0 +1,142 @@
+"""User detail based tables.
+
+Revision ID: 6a38281e6fcc
+Revises: 70bea50174bf
+Create Date: 2024-04-23 19:23:02.085782
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "6a38281e6fcc"
+down_revision = "70bea50174bf"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "users",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("glific_user_id", sa.Integer(), nullable=True),
+        sa.Column("phone", sa.String(length=50), nullable=False),
+        sa.Column("name", sa.String(length=255), nullable=True),
+        sa.Column("location", sa.String(length=255), nullable=True),
+        sa.Column("created_on", sa.DateTime(), nullable=False),
+        sa.Column("updated_on", sa.DateTime(), nullable=False),
+        sa.PrimaryKeyConstraint("id"),
+    )
```

Comment:
Let's add an index for user's phone for better query performance while performing a filter based on the phone.

```suggestion
    op.create_table(
        "users",
        sa.Column("id", sa.Integer(), nullable=False),
        sa.Column("glific_user_id", sa.Integer(), nullable=True),
        sa.Column("phone", sa.String(length=50), nullable=False),
        sa.Column("name", sa.String(length=255), nullable=True),
        sa.Column("location", sa.String(length=255), nullable=True),
        sa.Column("created_on", sa.DateTime(), nullable=False),
        sa.Column("updated_on", sa.DateTime(), nullable=False),
        sa.PrimaryKeyConstraint("id"),
    )
    op.create_index(op.f("ix_users_phone"), "users", ["phone"])
```

Reply:
Please add it in downgrade query as well